pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
```

Comment:
Shouldn't the span-related code in this function be conditional on whether tracing is enabled or not? This seems like a potentially large amount of work to be doing in the primary API dispatch loop.